runners/direct-java/pom.xml

```diff
@@ -83,7 +91,7 @@
               <failIfNoTests>true</failIfNoTests>
               <dependenciesToScan>
                 <dependency>org.apache.beam:beam-sdks-java-core</dependency>
-                <dependency>org.apache.beam:beam-runners-java-core</dependency>
+                <dependency>org.apache.beam:beam-runners-core-java</dependency>
```

Comment:
Doesn't seem to work, I made sure that there was a NeedsRunner annotation on a test and it never executed.